Packs/Zoom/Integrations/Zoom_IAM/Zoom_IAM.py

```diff
@@ -0,0 +1,291 @@
+
+    @staticmethod
+    def handle_exception(user_profile: IAMUserProfile,
+                         e: Union[DemistoException, Exception],
+                         action: IAMActions):
+        """ Handles failed responses from the application API by setting the User Profile object with the result.
+            The result entity should contain the following data:
+            1. action        (``IAMActions``)       The failed action                       Required
+            2. success       (``bool``)             The success status                      Optional (by default, True)
+            3. skip          (``bool``)             Whether or not the command was skipped  Optional (by default, False)
+            3. skip_reason   (``str``)              Skip reason                             Optional (by default, None)
+            4. error_code    (``Union[str, int]``)  HTTP error code                         Optional (by default, None)
+            5. error_message (``str``)              The error description                   Optional (by default, None)
+
+            Note: This is the place to determine how to handle specific edge cases from the API, e.g.,
+            when a DISABLE action was made on a user which is already disabled and therefore we can't
+            perform another DISABLE action.
+
+        :type user_profile: ``IAMUserProfile``
+        :param user_profile: The user profile object
+
+        :type e: ``Union[DemistoException, Exception]``
+        :param e: The exception object - if type is DemistoException, holds the response json object (`res` attribute)
+
+        :type action: ``IAMActions``
+        :param action: An enum represents the current action (GET, UPDATE, CREATE, DISABLE or ENABLE)
+        """
+        if isinstance(e, DemistoException) and e.res is not None:
+            error_code = e.res.status_code
+
+            if action == IAMActions.DISABLE_USER and error_code in ERROR_CODES_TO_SKIP:
+                skip_message = 'Users is already disabled or does not exist in the system.'
+                user_profile.set_result(action=action,
+                                        skip=True,
+                                        skip_reason=skip_message)
+
+            try:
+                resp = e.res.json()
+                error_message = get_error_details(resp)
+            except ValueError:
+                error_message = str(e)
+        else:
+            error_code = ''
+            error_message = str(e)
+
+        user_profile.set_result(action=action,
+                                success=False,
+                                error_code=error_code,
+                                error_message=error_message)
+
+        demisto.error(traceback.format_exc())
+
+
+'''HELPER FUNCTIONS'''
+
+
+def get_jwt(api_key: str, api_secret: str) -> str:
+    """
+    Encode the JWT token given the api ket and secret
+    """
+    tt = time.time()
```

Comment:
Let's give the variable a more descriptive name.
This will help go over the code more easily.